src/_help/customization-options/logo-meta-images.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Logo and Meta images
```

Comment:
We should add the "color" part to this page. So, maybe "Logo, Color and Meta image"? (do not forget to update the page name to reflect this change)